packages-oss.lock/pkgs.yml

```diff
@@ -1,15 +1,15 @@
 # ***
 # WARNING: Do not EDIT or MERGE this file, it is generated by packagespec.
 # ***
-lockid: 146a431e832db58c
+lockid: 48047ea24e91e50
 packagespec-version: 0.2.2
 cache-version: 10
 packages:
-  - packagespecid: a8bdeecb59f5f37dc83943709455b9fd277df5f8
+  - packagespecid: b6c12ff70f70cf013c61fafabe18ce98025cb6de
     inputs:
       BINARY_NAME: vault
       CGO_ENABLED: "0"
-      GO_VERSION: 1.16.2
+      GO_VERSION: 1.15.7
```

Comment:
It looks like some things in this file were reverted. Maybe a bad merge caused this?

Reply:
I think this is from a bad merge, yes. I re-re-re-ran `make packages` in 61e63b4d4 to fix this.